xclim/indices/run_length.py

```diff
@@ -618,7 +618,7 @@ def windowed_run_events_1d(arr: Sequence[bool], window: int):
     return (v * rl >= window).sum()
 
 
-def windowed_run_count_ufunc(x: Sequence[bool], window: int) -> xr.apply_ufunc:
+def windowed_run_count_ufunc(x: Sequence[bool], window: int) -> Callable:
```

Comment:
Euh, isn't the result a DataArray? I think we mistakenly wrote that those ufunc return "function" instead of datarrays.